crates/oxc_parser/src/diagnostics.rs

```diff
@@ -38,6 +38,48 @@ pub fn unexpected_token(span: Span) -> OxcDiagnostic {
     OxcDiagnostic::error("Unexpected token").with_label(span)
 }
 
+#[cold]
+pub fn merge_conflict_marker(
+    start_span: Span,
+    middle_span: Option<Span>,
+    end_span: Option<Span>,
+) -> OxcDiagnostic {
+    let mut diagnostic = OxcDiagnostic::error("encountered diff marker")
```

Comment:
```suggestion
    let mut diagnostic = OxcDiagnostic::error("Encountered diff marker")
```
other errors have the title capitialized.